### DISCUSSION TITLE
marmite/alt-hello-streams

### DISCUSSION THREAD

**Discussion by giscus** [rochacbruno/marmite#178] [Comments]
# marmite/alt-hello-streams

This content does not show on index page
and also does not show on main rss feed.
Because this conte...

https://rochacbruno.github.io/marmite/alt-hello-streams.html

**Comment by bgruszka** [comment]
Is it possible to have multilang site using streams or should I use alternative approach?

**Reply by rochacbruno** [reply]
I plan to use streams for it, for example: 

```yaml
---
title: Hello
stream: en
---
```

Then there will be a `/en.html` index page that I can link on the menu.

---

There is an issue about improving it https://github.com/rochacbruno/marmite/issues/154 

This issue will cover having a list of `languages` on the config file and then on each post you can link to translations.
